addons/point_of_sale/tests/test_pos_basic_config.py

```diff
@@ -1258,3 +1271,133 @@ def test_double_syncing_same_order(self):
         self.assertEqual(order.picking_count, 1, 'Order should have one picking')
         self.assertEqual(len(order.payment_ids), 1, 'Order should have one payment')
         self.assertEqual(self.env['account.move'].search_count([('ref', '=', order.name)]), 1, 'Order should have one invoice')
+
+    def test_copy_multiple_pos_configs(self):
+        """Copying multiple POS configs should duplicate each independently with valid payment methods."""
+
+        self.env['pos.payment.method'].search([]).unlink()
+
+        cash_pm1 = self.env['pos.payment.method'].create({
+            'name': 'Cash PM 1',
+            'journal_id': self.journal1.id,
+            'receivable_account_id': self.receivable.id,
+            'is_cash_count': True,
+            'company_id': self.env.company.id,
+        })
+        cash_pm2 = self.env['pos.payment.method'].create({
+            'name': 'Cash PM 2',
+            'journal_id': self.journal2.id,
+            'receivable_account_id': self.receivable.id,
+            'is_cash_count': True,
+            'company_id': self.env.company.id,
+        })
+        pos1 = self.env['pos.config'].create({
+            'name': 'POS Config 1',
+            'journal_id': self.journal1.id,
+            'invoice_journal_id': self.journal1.id,
+            'payment_method_ids': [(6, 0, [cash_pm1.id])],
+        })
+        pos2 = self.env['pos.config'].create({
+            'name': 'POS Config 2',
+            'journal_id': self.journal2.id,
+            'invoice_journal_id': self.journal2.id,
+            'payment_method_ids': [(6, 0, [cash_pm2.id])],
+        })
+
+        copied_configs = (pos1 + pos2).copy()
+
+        self.assertEqual(len(copied_configs), 2)
+        self.assertTrue(all(len(copy.payment_method_ids) == 1 for copy in copied_configs))
+        self.assertTrue(all(pm.is_cash_count for pm in copied_configs.mapped('payment_method_ids')))
```

Comment:
```suggestion
 def test_copy_multiple_pos_configs(self):
        """Copying multiple POS configs should duplicate each independently with valid payment methods."""
        config2 = self.basic_config.copy()
        copied_configs = (self.basic_config + config2).copy()

        self.assertEqual(len(copied_configs), 2)
        self.assertTrue(all(len(copy.payment_method_ids) == 1 for copy in copied_configs))
        self.assertTrue(all(pm.is_cash_count for pm in copied_configs.mapped('payment_method_ids')))
```
You should use existing data in your test case.
Apply same for all of your tests